Doc/library/ast.rst

```diff
@@ -729,7 +722,10 @@ Comprehensions
 
    .. doctest::
 
-        >>> print(ast.dump(ast.parse('[x for x in numbers]', mode='eval'), indent=4))
+        >>> print(ast.dump(
+        ...     ast.parse('[x for x in numbers]', mode='eval'),
+        ...     indent=4, show_empty=True,
```

Comment:
What is the logic behind which docs (doctests) here use `show_empty=True` and which don't?

I would expect none of them to use it, except for maybe one example demonstrating `show_empty=True` itself, where we document the argument.

Reply:
Hm, indeed. But, I think that we do the opposite. Since this is the place where every node is documented, we need to show all properties of all nodes. It would be strange to hide them here. And `show_empty=True` will also be shown.

Reply:
Yeah, I've tried my suggestion and didn't like it. Now I agree with you, all should be minimized.